pkgs/development/python-modules/pygls/default.nix

```diff
@@ -23,8 +28,8 @@ buildPythonPackage rec {
   checkPhase = "pytest";
```

Comment:
Can you convert this to use `pytestCheckHook` instead? This way we can drop the checkPhase overwrite.

 https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pytestcheckhook

Reply:
Yep, I will thank you! I didn't know about `pytestCheckHook`, it looks like it was recently added.